x-pack/platform/plugins/shared/security/server/authentication/providers/saml.test.ts

```diff
@@ -1674,4 +1675,163 @@ describe('SAMLAuthenticationProvider', () => {
   it('`getHTTPAuthenticationScheme` method', () => {
     expect(provider.getHTTPAuthenticationScheme()).toBe('bearer');
   });
+
+  describe('UIAM mode', () => {
```

Comment:
Both token refresh and invalidation aren't yet supported by the UIAM, so I'm not paying a lot of attention to testing these scenarios, but I'll cover these with tests during next stage when these API are available to us.